app/core/models/ocelot/stanzas/SequenceStanza.scala

```diff
@@ -52,40 +52,70 @@ object SequenceStanza {
     )(unlift(SequenceStanza.unapply))
 }
 
-case class Sequence(text: Phrase,
-                    override val next: Seq[String],
-                    options: Seq[Phrase],
-                    label: Option[String],
-                    stack: Boolean) extends VisualStanza with Populated with DataInput {
-  override val labelRefs: List[String] = labelReferences(text.english) ++ options.flatMap(a => labelReferences(a.english))
-  override val labels: List[Label] = label.fold[List[Label]](Nil)(l => List(ScalarLabel(l)))
+trait Sequence extends VisualStanza with Populated with DataInput {
```

Comment:
think your original idea of placing the exclusive option at head of list and sticking with a single Sequence class was simpler and clearer